tasks/transform_conformance/tests/babel-plugin-transform-async-to-generator/test/fixtures/arguments/assign/input.js

```diff
@@ -0,0 +1,4 @@
+const ArgumentsAssignment = async () => {
+  let arguments = arguments;
+  console.log(arguments);
+};
```

Comment:
This is quite a weird example. It'd be an immediate error to call this function because `let arguments = arguments;` is a TDZ violation.

Is it possible to make a more reasonable example which still tests what it's meant to test?

Reply:
Oh, yeah, I just realized this.